### PR TITLE
.rubocop.yml: replace Style/TrailingComma

### LINKED ISSUE #48105: Rubocop uses now-removed cop `Style/TrailingComma`
As of rubocop [0.36](https://github.com/bbatsov/rubocop/releases/tag/v0.36.0), the cop `Style/TrailingComma` no longer exists.

Running `brew audit` or `brew style` on any formula (eg. `brew style maven`) after doing a `gem update` gives the following error message:

> Error: The `Style/TrailingComma` cop no longer exists. Please use `Style/TrailingCommaInLiteral`
> and/or `Style/TrailingCommaInArguments` instead.
> (configuration found in /home/bob/.linuxbrew/Library/.rubocop.yml)

(Note: you might notice from the above message that I'm using Linuxbrew rather than Mac-Homebrew. But I've confirmed by looking at your current .rubocop.yml file that it has the same issue, and believe this should be fixed here rather than at Linuxbrew).

### PR DESCRIPTION
**Pull request by rwhogg**
Closes #48105

### CODE REVIEW

File: Library/.rubocop.yml [hunk 1]
```diff
@@ -125,5 +125,7 @@ Style/DotPosition:
   EnforcedStyle: trailing
 
 # makes diffs nicer
-Style/TrailingComma:
+Style/TrailingCommaInArguments:
+  EnforcedStyleForMultiline: comma
```
Comment (xu-cheng) [inline Library/.rubocop.yml]: We probably want `no_comma` for this. As trailing comma for arguments is syntax error for Ruby 1.8.

**Comment by xu-cheng** [comment]
Would you please also update the rubocop version in https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/style.rb#L11 

Thanks.

File: Library/.rubocop.yml (hunk 1 above)
Comment (rwhogg) [inline Library/.rubocop.yml]: Gotcha, will do

On January 16, 2016 12:02:00 AM PST, Xu Cheng <notifications@github.com> wrote:
>> @@ -125,5 +125,7 @@ Style/DotPosition:
>>    EnforcedStyle: trailing
>>  
>>  # makes diffs nicer
>> -Style/TrailingComma:
>> +Style/TrailingCommaInArguments:
>> +  EnforcedStyleForMultiline: comma
>
>We probably want `no_comma` for this. As trailing comma for arguments
>is syntax error for Ruby 1.8.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/Homebrew/homebrew/pull/48144/files#r49928971

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by rwhogg** [comment]
No problem

On January 16, 2016 12:02:39 AM PST, Xu Cheng <notifications@github.com> wrote:
>Would you please also update the rubocop version in
>https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/cmd/style.rb#L11
>
>
>Thanks.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/Homebrew/homebrew/pull/48144#issuecomment-172170700

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by xu-cheng** [comment]
Thanks for the pull request! 🎉 Homebrew depends on contributions from community members like you and we're grateful for your support.

**Pull request closed without merging**